Gemfile.lock

```diff
@@ -88,6 +88,7 @@ GEM
 
 PLATFORMS
   arm64-darwin-20
+  x86_64-linux
 
```

Comment:
should there be two platforms listed here?

Reply:
Yup. `x86_64-linux` is the environment CI runs, while `arm64-darwin-20` is my environment.

Reply:
👍